sql/testdata/aggregate

```diff
@@ -6,6 +6,17 @@ CREATE TABLE kv (
   s STRING
 )
 
+# Aggregate functions return NULL if there are no rows.
+query IIIRRRBB
+SELECT MIN(1), MAX(1), COUNT(1), AVG(1), STDDEV(1), VARIANCE(1), BOOL_AND(true), BOOL_AND(false) FROM kv
```

Comment:
Indeed. Moreover, there were two extra use cases missing from the tests too. done.